conda-forge.yml

```diff
@@ -1,3 +1,9 @@
+azure:
+  free_disk_space: true
+  settings_linux:
+    swapfile_size: 10GiB
```

Comment:
Still seeing the same error. Not sure what we should bump this too. Any thoughts?

Reply:
I'm really unsure where there's a lot of "weird environment" in the loop here. The fact that different components are failing with signal 11 points to environment from my perspective.

Reply:
IIUC the feedstock is using emulation for arch builds. Could that be an issue? Should we use cross-compilation or native builds?

https://github.com/conda-forge/librmm-feedstock/blob/f0423fbbaad03e68341ca521ce42b5e080417f9a/conda-forge.yml#L9-L11

Reply:
This is only building tests to ensure dependency version changes don't break downstream usage of RMM headers. I would prefer to keep the setup as simple as possible where the easiest path forward would possibly be to just only build the tests on linux-64?

Reply:
That sounds ok to me